lib/rubyapi_rdoc_generator.rb

```diff
@@ -86,6 +86,7 @@ def generate_objects
         superclass: superclass,
         included_modules: doc.includes.map(&:name),
         constants: doc.constants.each_with_object([]) { |c, arr| arr << {name: c.name, description: clean_description(doc.full_name, c.description)} },
+        attributes: doc.attributes.each_with_object([]) { |a, arr| arr << {name: a.name, description: clean_description(doc.full_name, a.description), access: readwrite_string(a.rw)} },
```

Comment:
Any reason these aren't just `map`s?

```suggestion
        constants: doc.constants.map { |c| {name: c.name, description: clean_description(doc.full_name, c.description)} },
        attributes: doc.attributes.map { |a| {name: a.name, description: clean_description(doc.full_name, a.description), access: readwrite_string(a.rw)} },
```

Reply:
Ah this is an artefact when I had an alternative design